Source/WebApi.HypermediaExtensions/DefaultHypermediaRelations.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
```

Comment:
The client can not access it here.

Reply:
In this case it's not meant for the client but for the server package as there are references to the relations.